turutan/turutan-mobile-app
# feature/TUR-35 - Turutan Mobile Menu Items Widget

## Purpose
Added home screen wireframe and scrollable modal bottom sheet menu

## Types of changes

What types of changes does your code introduce?

## How to Test
1. run and build the apps to see UI
2. Tap on "Lainnya"

## Environment

<details><summary> Attach your `flutter doctor -v` inside code block: </summary>
<p>

```
[✓] Flutter (Channel stable, 2.2.3, on macOS 11.4 20F71 darwin-x64, locale en-ID)
    • Flutter version 2.2.3 at /Users/ittcel/fvm/versions/stable
    • Framework revision f4abaa0735 (6 weeks ago), 2021-07-01 12:46:11 -0700
    • Engine revision 241c87ad80
    • Dart version 2.13.4

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /Users/ittcel/Library/Android/sdk
    • Platform android-30, build-tools 30.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6916264)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.5.1, Build version 12E507
    • CocoaPods version 1.10.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6916264)

[✓] IntelliJ IDEA Community Edition (version 2020.3.4)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart

[✓] VS Code (version 1.59.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.25.0

[✓] Connected device (2 available)
    • iPhone 11 (mobile) • CD571557-9E5A-40B3-B4A9-C9AA84131FD5 • ios            • com.apple.CoreSimulator.SimRuntime.iOS-14-5 (simulator)
    • Chrome (web)       • chrome                               • web-javascript • Google Chrome 92.0.4515.131

• No issues found!
```

</p>
</details>

## Expected Results
Modal Bottom Menu will be opened from bottom to Top

## Screenshot (if appropriate)
![Simulator Screen Recording - iPhone 11 - 2021-08-22 at 09 14 45](https://user-images.githubusercontent.com/24664153/130339719-7e4a028a-51e0-4c6c-a3bf-498c3b2e9ee3.gif)
